### DISCUSSION TITLE
Warning message after 08b

### DISCUSSION THREAD

**Discussion by hannah-e** [genetics-of-dna-methylation-consortium/godmc_phase2#76] [Q&A]
module 08b runs successfully, but I get a warning message. Please let me know if I should try different parameters!
Here is the message:

Warning message:
In Matrix_eQTL_main(snps = snps, gene = gene, cvrt = cvrt, output_file_name = output_file_name,  :
  Warning: pvOutputThreshold may be too large.
Expected number of findings > 6607916
Successfully completed inversion meQTL analysis

**Comment by yocra3** [comment]
Hi Hannah,

This is correct. I set the p-value threshold to 1 so all the associations are reported. The function returned all the associations (as expected) and it considers that is reporting too many associations and suggests to lower the p-value. You can safely ignore this warning.

Bests,
Carlos Ruiz

**Reply by hannah-e** [reply]
@yocra3 Thank you. Once the sftp issue is resolved (#70), I will upload the results. Best wishes, Hannah

**Reply by yocra3** [reply]
Could you give a try to uploading the results? I have not been able to reproduce the problem and other researchers tried to upload the files and found other issues. I would be very helpful if you try it on your side and you send me the log file.

**Reply by hannah-e** [reply]
@yocra3 I sent you an email with the details. Best, Hannah
